Comments/ClientApp/src/components/CommentList/Drawer.scss

```diff
@@ -1,5 +1,10 @@
-@import "../../../node_modules/@nice-digital/design-system/src/stylesheets/nice-design-system";
-@import "../../styles/accessibility.scss";
+//@import "../../../node_modules/@nice-digital/design-system/src/stylesheets/nice-design-system";
```

Comment:
Mega pedantic but please could you remove the commented out stuff if it's no longer needed? Ta